utilities/score/entrypoint.py

```diff
@@ -0,0 +1,62 @@
+import os
+import six
+import re
+import time
+import random
+import subprocess
+import json
+
+from nmtwizard.utility import Utility
+from nmtwizard.logger import get_logger
+from nmtwizard.utility import resolve_environment_variables
+from nmtwizard.storage import StorageClient
+
+logger = get_logger(__name__)
+
+class ScoreUtility(Utility):
+
+    def __init__(self):
+        super(ScoreUtility, self).__init__()
+        self._tools_dir = os.getenv('TOOLS_DIR', '/root/tools')
+
+    @property
+    def name(self):
+        return "score"
+
+    def declare_arguments(self, parser):
+        subparsers = parser.add_subparsers(help='Run type', dest='cmd')
+        parser_score = subparsers.add_parser('score', help='Evaluate translation.')
+        parser_score.add_argument('-i', '--input', required=True, nargs='?',
```

Comment:
Why `nargs='?'`? I think it can be removed.

Reply:
done